src/main/java/taxi/service/DriverServiceImpl.java

```diff
@@ -35,4 +36,10 @@ public Driver update(Driver driver) {
     public boolean delete(Long id) {
         return driverDao.delete(id);
     }
+
+    @Override
+    public Driver findByLogin(String login) {
+        return driverDao.findByLogin(login)
+                .orElseThrow(() -> new NoSuchElementException("Login or password was incorrect"));
+    }
```

Comment:
check situation when you try to login with non-existing login
what will happen? 
do you see any useful message about that login is incorrect?